File: 2137-final-value-of-variable-after-performing-operations/2137-final-value-of-variable-after-performing-operations.py
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        ans = 0
        for operation in operations:
            if operation == "--X" or operation == "X--":
                ans -= 1
            elif operation == "++X" or operation == "X++":
                ans += 1
        return ans
    

File: 2137-final-value-of-variable-after-performing-operations/2137-final-value-of-variable-after-performing-operations.cs
public class Solution {
    public int FinalValueAfterOperations(string[] operations) {
        int ans = 0;
        foreach (string operation in operations) {
            if (operation == "--X" || operation == "X--") {
                ans--;
            } else if (operation == "++X" || operation == "X++") {
                ans++;
            }
        }
        return ans;
    }
}